src/general/MainMenu.cs

```diff
@@ -68,15 +66,19 @@ public class MainMenu : NodeWithInput
     private Control creditsContainer = null!;
     private CreditsScroll credits = null!;
     private LicensesDisplay licensesDisplay = null!;
-
-    private Button newGameButton = null!;
     private Button freebuildButton = null!;
 
     private Label storeLoggedInDisplay = null!;
 
     private CustomConfirmationDialog gles2Popup = null!;
     private ErrorDialog modLoadFailures = null!;
 
+    public static void OnEnteringGame()
```

Comment:
I'd make this a non-static method to be able to put this later in the class where the other new game methods are, to not have this be the first piece of code someone sees when they try to read the main menu file.

Reply:
Okay, I read now this entire file... I guess it's not worth trying to think of a different design for this as this is needed to be called from the new game menu as there's other stuff to fix in this PR...